backend/events/migrations/0002_auto_20190415_1538.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2 on 2019-04-15 15:38
```

Comment:
These migrations are duplicative. Please migrate down, delete all of them from 0002 to 0006, and recreate them as a single migration.